chapters/appendix-IV-SPDX-license-expressions.md

```diff
@@ -1,175 +1,174 @@
 
-Sometimes a set of license terms apply except under special circumstances. In this case, use the binary "WITH" operator to construct a new license expression to represent the special exception situation. A valid \<license-expression> is where the left operand is a \<simple-expression> value and the right operand is a \<license-exception-id> that represents the special exception terms.
+Parentheses (`()`) can be used to specify an explicit grouping to override [operator precedence](#IV.2.6).  This is similar to the use of parentheses in algebraic expressions like `(5+7)/2`.
 
-For example, when the Bison exception is to be applied to GPL-2.0+, the expression would be:
+For instance, the following expression:
 
-    (GPL-2.0+ WITH Bison-exception-2.2)
+    MIT AND (LGPL-2.1+ OR BSD-3-Clause)
 
-The current set of valid exceptions can be found in [Appendix I, section 2](appendix-I-SPDX-license-list.md#I.2). For the most up to date set of exceptions please see [spdx.org/licenses](https://spdx.org/licenses). If the applicable exception is not found on the SPDX License Exception List, then use a single \<license-ref> to represent the entire license terms (including the exception).
+represents a license requring both `MIT` and the expression `LGPL-2.1+ OR BSD-3-Clause`, because the parenthesis require the enclosed expression take precedence over (is applied before) the `AND` operator.
 
-### 4) Order of Precedence and Parentheses
+### IV.2.6 Operator Precedence <a name="IV.2.6"></a>
 
-The order of application of the operators in an expression matters (similar to mathematical operators). The default operator order of precedence of a \<license-expression> a is:
+The operators described above have the following precedence, from highest (binding tightest) at the top, to lowest (loosest) at the bottom:
 
+    Grouping
     +
     WITH
     AND
     OR
 
-where a lower order operator is applied before a higher order operator.
-
 For example, the following expression:
 
-    LGPL-2.1 OR BSD-3-Clause AND MIT 
+    LGPL-2.1 OR BSD-3-Clause AND MIT
 
-represents a license choice between either LGPL-2.1 and the expression BSD-3-Clause AND MIT because the AND operator takes precedence over (is applied before) the OR operator.
+represents a license choice between either `LGPL-2.1` and the expression `BSD-3-Clause AND MIT` because the `AND` operator takes precedence over (is applied before) the `OR` operator.
 
-When required to express an order of precedence that is different from the default order a \<license-expression> can be encapsulated in pairs of parentheses: ( ), to indicate that the operators found inside the parentheses takes precedence over operators outside. This is also similar to the use of parentheses in an algebraic expression e.g., (5+7)/2.
+## IV.3 License Expressions in XML <a name="IV.3"></a>
 
-For instance, the following expression:
+### IV.3.1 The `+` Operator <a name="IV.3.1"></a>
+
+FIXME: How are you supposed to cover this in XML?
```

Comment:
The following text can be added:
The `+` operator can be expressed in XML via a `<spdx:OrLaterOperator>` element, with a single `spdx:member` property for the license preceeding the `+` operator.

     <spdx:OrLaterOperator>
          <spdx:member rdf:resource="http://spdx.org/licenses/GPL-2.0"/>
     </spdx:OrLaterOperator>

Reply:
Addressed in cf8decf → 9c8d189.